applications/stepswitch/src/stepswitch_resources.erl

```diff
@@ -152,7 +153,9 @@ endpoints(Number, OffnetJObj) ->
 -spec maybe_get_endpoints(ne_binary(), kapi_offnet_resource:req()) -> kz_json:objects().
 maybe_get_endpoints(Number, OffnetJObj) ->
     case kapi_offnet_resource:hunt_account_id(OffnetJObj) of
-        'undefined' -> get_global_endpoints(Number, OffnetJObj);
+        'undefined' ->
+            lager:debug("attempting to find global resources"),
+            get_endpoints(get(), Number, OffnetJObj);
```

Comment:
This `get` is `stepswitch_resources:get/0` not `erlang:get/0`.
Should it be ranamed?